.codespellrc

```diff
@@ -16,5 +16,6 @@
 # under the License.
 
 [codespell]
+dictionary = .codespell-dictionary,-
```

Comment:
Note: `-` is the default (builtin) dictionary.
We need this to add dictionaries, instead of replacing them.